BetaCameras/Kinect4Azure/Kinect4Azure.cs

```diff
@@ -109,13 +98,13 @@ public K4AColorResolution ColorResolution
             }
         }
 
-        ListParamDesc<K4AColorResolution> ColorResolutionDesc
+        private ListParamDesc<K4AColorResolution> ColorResolutionDesc
         {
             get
             {
                 ListParamDesc<K4AColorResolution> res = new ListParamDesc<K4AColorResolution>(typeof(K4AColorResolution))
                 {
-                    Description = "Resolution of the Color Image",
+                    Description = "Resolution of the Color channel",
```

Comment:
```suggestion
                    Description = "Resolution of the color channel",
```

Reply:
I'd rather keep the writing in sync with `ChannelNames`. So what about this?
```suggestion
                    Description = $"Resolution of the {ChannelNames.Color} channel",
```

Reply:
Why then not using a capital letter for the `C` in channel as well (both `color` and `channel` are nouns).